bridge/engine/AudioForwarderReceiveJob.cpp

```diff
@@ -188,34 +330,43 @@ void AudioForwarderReceiveJob::run()
         }
     }
 
-    const auto oldRolloverCounter = _ssrcContext.lastUnprotectedExtendedSequenceNumber >> 16;
-    const auto newRolloverCounter = _extendedSequenceNumber >> 16;
-    if (newRolloverCounter > oldRolloverCounter)
+    if (!unprotect(*_packet))
     {
-        logger::debug("Setting new rollover counter for ssrc %u", "AudioForwarderReceiveJob", _ssrcContext.ssrc);
-        if (!_sender->setSrtpRemoteRolloverCounter(_ssrcContext.ssrc, newRolloverCounter))
-        {
-            logger::error("Failed to set rollover counter srtp %u, mixer %s",
-                "AudioForwarderReceiveJob",
-                _ssrcContext.ssrc,
-                _engineMixer.getLoggableId().c_str());
-            return;
-        }
+        return;
     }
 
-    if (!_sender->unprotect(*_packet))
+    int calculatedAudioLevel = -1;
+    if (_hasMixedAudioStreams && _ssrcContext.rtpMap.format == bridge::RtpMap::Format::OPUS)
     {
-        logger::error("Failed to unprotect srtp %u, mixer %s",
-            "AudioForwarderReceiveJob",
-            _ssrcContext.ssrc,
-            _engineMixer.getLoggableId().c_str());
-        return;
+        calculatedAudioLevel = decodeOpus(*_packet, !audioLevel.isSet());
+    }
+    else if (_ssrcContext.rtpMap.format == bridge::RtpMap::Format::OPUS && _needAudioLevel && !audioLevel.isSet())
+    {
+        calculatedAudioLevel = computeOpusAudioLevel(*_packet);
```

Comment:
If we have to make this feature configurable i guess we need to do something here to emulate the audio level on ptt.